### ISSUE TITLE
[Question]How to configure eslint to report template to use ref directly instead of using ref.value?

### ISSUE THREAD

**Issue by yoyo837** [vuejs/eslint-plugin-vue#2300]

**Comment by FloEdelmann** [comment]
Please specify your question in more detail, it's not clear from the title alone. Some example code would help immensely. In the meantime, I'll close this issue, but feel free to comment here again.

**Comment by yoyo837** [comment]
Q1: I wish there was a rule reporting that my first usage (`a`) is incorrect.
```vue
<script setup>
const useIds = ref()
</script>
<template>
  <!-- a -->
  {{ useIds.value }}
  <!-- b -->
  {{ useIds }}
</template>
```

**Comment by yoyo837** [comment]
Q2: And I wish there was a rule reporting that my first usage (`c`) is incorrect.
```vue
<script setup>
const useIds = ref()
// c
console.log(useIds)
// d
console.log(useIds.value)
</script>
```

**Comment by yoyo837** [comment]
Looking forward to your help. Thanks!

**Comment by FloEdelmann** [comment]
Q1: There is a similar rule: [`vue/this-in-template`](https://eslint.vuejs.org/rules/this-in-template.html). Could you please open a new issue following the rule proposal template to add a new rule or adjust this rule?

Q2: There is the [`vue/no-ref-as-operand`](https://eslint.vuejs.org/rules/no-ref-as-operand.html) rule, which prevents similar cases. Passing a ref to a function without dereferencing it is however often what you need to not lose reactivity (see also [`vue/no-ref-object-reactivity-loss`](https://eslint.vuejs.org/rules/no-ref-object-reactivity-loss.html)). So this rule is basically impossible, because it can not know whether you want to dereference or not i a particular case.

**Comment by yoyo837** [comment]
Thanks for your reply. I am now enlightened. I will rethink what I want.
